deb/Makefile

```diff
@@ -45,49 +45,49 @@ debian: debian-stretch debian-jessie ## build all debian deb packages
 raspbian: raspbian-stretch debian-jessie ## build all raspbian deb packages
 
 .PHONY: ubuntu-xenial
-ubuntu-xenial: engine-$(ARCH).tar ## build ubuntu xenial deb packages
```

Comment:
Ideally these changes would be in a separate request. 
(These aren't RPM changes)